### DISCUSSION TITLE
Get all downstream properties

### DISCUSSION THREAD

**Discussion by helgestein** [emmo-repo/EMMO-python#508] [Q&A]
Didn't want to create an issue and Q&A seemed more appropriate.

I would like to get all the properties that define something to create sort of a coverage percentage of what I was supplied by the user.
What I have is this:
```
from ontopy import get_ontology
battinfo = get_ontology('https://raw.githubusercontent.com/BIG-MAP/BattINFO/master/battinfo.ttl').load()
electrolyte = battinfo.Electrolyte
```

How can I get all properties that would unequivocally define an electrolyte?

**Comment by jesper-friis** [comment]
There are separate methods/properties for different aspects:

  - Annotations: `electrolyte.get_annotations()`
  -  Restrictions (incl superclasses): `electrolyte.is_a`
  -  Equivalent expressions: `electrolyte.equivalent_to`
  -  Disjoint classes: `electrolyte.disjoints()`
  - Declared disjoint unions: `electrolyte.disjoint_unions`
